system/users.nix

```diff
@@ -16,5 +16,9 @@ in {
         description = "Thomas";
         extraGroups = [ "wheel" ];
         hashedPassword = "$6$rounds=2000000$htFKKf65jcKCw09Z$JNmYnL5lIBZP6dvqYXUmj0vzzaiRteXOwlJzkcYcRCYdT5Zt8TVJWvtT4w4Q8suBneVOLEjxsMIf0yEY4BDrz1";
+        packages = with pkgs; [
+            lynx
+            vim
+        ];
```

Comment:
This might as well go in `environment.systemPackages`, so that all users can use these if desired.